source/Azure.Common/Extensions/PropertyDictionaryExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+namespace Sashimi.Azure.Common.Extensions
+{
+    public static class PropertyDictionaryExtensions
+    {
+        public static bool ContainsPropertyWithValue(this IDictionary<string, string> dictionary, string key)
```

Comment:
Consider renaming this global extension to:
```suggestion
        public static bool ContainsPropertyWithNonEmptyValue(this IDictionary<string, string> dictionary, string key)
```